src/components/settings/SettingsSSOTab/steps/SSOConfigIDPStep.test.jsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { Provider } from 'react-redux';
+import SSOConfigIDPStep from './SSOConfigIDPStep';
+import { SSOConfigContextProvider, SSO_INITIAL_STATE } from '../SSOConfigContext';
+import { getMockStore, initialStore } from '../testutils';
```

Comment:
these utils could arguable be elevated even higher up and used in all the admin portal tests, but I did not want to go too far rn